pkg/dogstatsd/parse.go

```diff
@@ -193,6 +207,11 @@ func (p *parser) parseFloat64List(rawFloats []byte) ([]float64, error) {
 	return values, nil
 }
 
+// extractContainerID parses the value of the container ID field.
+func (p *parser) extractContainerID(rawContainerIDField []byte) string {
+	return string(rawContainerIDField[len(containerIDFieldPrefix):])
```

Comment:
Not a big problem, but perhaps we could keep containerID as a `[]byte` here, until we concatenate it with the prefix. This will avoid an intermediate copy of container id.